lib/http/auth/basic.go

```diff
@@ -112,6 +112,7 @@ func CustomAuth(fn CustomAuthFn, realm string) httplib.Middleware {
 				}
 				if value != nil {
 					r = r.WithContext(context.WithValue(r.Context(), ContextAuthKey, value))
+					next.ServeHTTP(w, r)
```

Comment:
I think this needs to go after the `}` so it always gets called, but it looks like the right fix.